edweiss-app/model/lectures/lectureDoc.ts

```diff
@@ -10,7 +10,6 @@
 
 import { FunctionFolder, FunctionOf } from '../functions';
 import Quizzes from '../quizzes';
-import { Timestamp } from '../time';
```

Comment:
Yes but it is still not used Timestamp in the model of lectureDoc during this PR

Reply:
Yeah I know, it was just a small remark so that when you re-introduce it later, you remember to use this `Timestamp` not the one directly provided by `firebase`. Hopes that clearer this way ?